src/CameraControls.ts

```diff
@@ -500,6 +500,21 @@ export class CameraControls extends EventDispatcher {
 		const lastDragPosition = new THREE.Vector2() as _THREE.Vector2;
 		const dollyStart = new THREE.Vector2() as _THREE.Vector2;
 
+		const disposePointer = ( pointer: PointerInput ) => {
+
+			const isPointerLockActive = this._domElement && this._domElement.ownerDocument.pointerLockElement === this._domElement;
+
+			if ( isPointerLockActive && this._activePointers.length <= 1 ) {
+
+				// NOTE Do not dispose pointer-locked pointer
+				return;
+
+			}
```

Comment:
Would we need this? It seems this does not happen.

Reply:
That's the check mentioned earlier - when you lock pointer, then press mouse1 a few times and drag around, the `endDragging` will dispose `pointermove` event listener and make pointerlock not work again.

Reply:
I understand.
Could we check the pointer before calling disposePointer?

Reply:
Kinda defeats the purpose of disposePointer 🥲

Reply:
I'm sorry, no offense meaning...🥲
I thought, `disposePointer()` should forcibly dispose of the pointer in any case.
And now `this._activePointers.splice(...)` has a name as `disposePointer()` which is better readability than mine...

maybe we could change the name of the function if it doesn't dispose of in some cases.

Reply:
No worries - would you like to push the changes yourself? I just wanted to add a suggestion to handle pointer lock - all the stylistic changes in the code I can leave up to you - you can push right to this PR and we merge 😌

Reply:
@mjurczyk sorry for the delay, and thanks for understanding. Okay, I will make a PR to yours!